flake8 error on master branch, file compose/bundle.py

When trying to run the test suite on master branch the pre-commit hook fails with an flake8 error:

```
$ script/test/default 
Flake8...................................................................Failed
hookid: flake8

compose/bundle.py:124:5: E722 do not use bare except'
```

--- a/compose/bundle.py
+++ b/compose/bundle.py
@@ -121,7 +121,7 @@ def get_image_digest(service, allow_push=False):
 def push_image(service):
     try:
         digest = service.push()
-    except:
+    except Exception:
         log.error(
             "Failed to push image for service '{s.name}'. Please use an "
             "image tag that can be pushed to a Docker "

